components/Creation/CreationComments/CreationComments.vue

```diff
@@ -11,7 +11,7 @@
 	const currentPage = ref(1); // 当前页码
 	const pinned = ref(false);
 	const search = ref("");
-	const pageCount = computed(() => Math.floor(commentsCount.value / pageSize) + 1);
+	const pageCount = computed(() => Math.ceil(commentsCount.value / pageSize));
```

Comment:
`Math.ceil(0 / 20)` 仍会导致报错。
```suggestion
	const pageCount = computed(() => Math.max(1, Math.ceil(commentsCount.value / pageSize)));
```

Reply:
我就知道这事没这么简单...